pages/badge/badge.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+*, ::before, ::after {
+    box-sizing: border-box;
+    margin: 0px;
+    padding: 0px;
+}
+
+body {
+    font-family: Poppins, sans-serif;
+    background-color: rgb(0, 0, 0);
+    color: rgb(255, 255, 255);
+}
+
+.badge-container{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 10rem;
+    align-items: center;
+    justify-content: center;
+    margin-top: 2rem;
+}
+
+.badge-notification, .badge-mail, .badge-cart, .badge-avatar{
+    position: relative;
+    display: inline-block;
+}
+
+/* badge-notification */
+.badge-notification>i{
+    font-size: 4rem;
+}
+
+.badge-notification-count{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    right: 0;
+    top: 0;
+    background:rgb(0, 153, 255);
+    color:rgb(255, 255, 255);
+    width: 1.7rem;
+    border: 2px solid black;
+    border-radius: 50%;
+}
+
+/* badge-mail */
+.badge-mail>i{
+    font-size: 4rem;
+}
+
+.badge-mail-count{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    right: -12px;
+    top: 0;
+    background:rgb(0, 153, 255);
+    color:rgb(255, 255, 255);
+    width: 1.7rem;
+    border: 2px solid black;
+    border-radius: 50%;
+}
+
+/* badge-cart */
+.badge-cart>i{
+    font-size: 4rem;
+}
+
+.badge-cart-count{
```

Comment:
Suggestion: badge-mail-count and badge-cart-count has common styling. you can use a common class name instead of two class names